patches/minecraft/net/minecraft/entity/EntityLivingBase.java.patch

```diff
@@ -47,15 +63,40 @@
                  while (i > 0)
                  {
                      int j = EntityXPOrb.func_70527_a(i);
-@@ -442,6 +445,7 @@
+@@ -442,6 +449,7 @@
      {
          this.field_70755_b = p_70604_1_;
          this.field_70756_c = this.field_70173_aa;
 +        net.minecraftforge.common.ForgeHooks.onLivingSetAttackTarget(this, p_70604_1_);
      }
  
      public EntityLivingBase func_110144_aD()
-@@ -670,8 +674,10 @@
+@@ -536,6 +544,7 @@
+         }
+ 
+         p_70014_1_.func_74757_a("FallFlying", this.func_184613_cA());
++        p_70014_1_.func_74776_a("LastHealth", this.getLastHealth());
+     }
+ 
+     public void func_70037_a(NBTTagCompound p_70037_1_)
+@@ -567,7 +576,7 @@
+         {
+             this.func_70606_j(p_70037_1_.func_74760_g("Health"));
+         }
+-
++        
```

Comment:
wat?

Reply:
I'm unsure what you are asking, I am simply trying my best to make the follow the given specifications and use genPatches to generate the patches.

Reply:
You have a change here you need to get rid of, its just adding indentation.